Integrations/Cybereason/Cybereason_test.py

```diff
@@ -0,0 +1,261 @@
+            "elementValues": {
+                "ownerMachine": {
+                    "totalValues": 1,
+                    "totalSuspicious": 0,
+                    "guessedTotal": 0,
+                    "totalMalicious": 0,
+                    "elementValues": [
+                        {
+                            "guid": "1899624444.1198775089551234567",
+                            "hasSuspicions": 'false',
+                            "elementType": "Machine",
+                            "name": "desktop-p0m5vad",
+                            "hasMalops": 'false'
+                        }
+                    ]
+                }
+            },
+            "malicious": 'false',
+            "filterData": {
+                "sortInGroupValue": "1899624463.3738670480412115128",
+                "groupByValue": "FileHashRuntime:0.4272124792099163455 "  # disable-secrets-detection
+            },
+            "suspicions": {
+                "reportedByAntiMalwareSuspicion": 1573393767860
+            },
+            "labelsIds": 'null',
+            "suspect": 'true',
+            "guidString": "1899624444.1198775089551234567",
+            "simpleValues": {
+                "sha1String": {
+                    "totalValues": 1,
+                    "values": [
+                        "984e5a25910edafd4567c0f51c6f2d779530451d"
+                    ]
+                },
+                "elementDisplayName": {
+                    "totalValues": 1,
+                    "values": [
+                        "bdata.bin"
+                    ]
+                },
+                "correctedPath": {
+                    "totalValues": 1,
+                    "values": [
+                        "c:\\windows\\temp\\sb-sim-temp-ymrc7u\\sb_5313555_bs_jwnggm\\bdata.bin"  # disable-secrets-detection
+                    ]
+                },
+                "md5String": {
+                    "totalValues": 1,
+                    "values": [
+                        "4778901e54f55d54435b2626123456a8"
+                    ]
+                },
+                "maliciousClassificationType": {
+                    "totalValues": 1,
+                    "values": [
+                        "av_detected"
+                    ]
+                }
+            }
+        }
+    }
+}
+
+params = {
+    'server': 'https://integration.cybereason.net:8443',
+    'credentials': {'credentials': {'sshkey': 'shelly'}},
+    'proxy': True}
+
+
+def test_query_file(mocker):
+    mocker.patch.object(demisto, 'params', return_value=params)
+    mocker.patch.object(demisto, 'args', return_value={'file_hash': '4778901e54f55d54435b2626923054a8'})
+    mocker.patch('Cybereason.client_certificate', side_effect=lambda: None, autospec=False)
+    mocker.patch('Cybereason.get_file_machine_details', return_value=MACHINE_OUTPUTS)
+    mocker.patch('Cybereason.query_file', return_value=FILE_OUTPUTS)
+    mocker.patch.object(demisto, 'results')
+    import Cybereason
```

Comment:
remove the import from here, and do it at the top of the code at the import section as follows:
`from Cybereason import query_file_command`

Reply:
Can't change that now since I'm mocking this object, so it's won't work

Reply:
@Itay4 As @Shellyber said, I don't think we have another option here. Will merge like this if we will find something else will change after.